test/integration/post-metadata.js

```diff
@@ -0,0 +1,61 @@
+/*eslint-env node, mocha */
+import uuid from 'uuid';
+import expect from '../../test-helpers/expect';
+import app from '../../index';
+import { serialize } from 'cookie';
+
+import initBookshelf from '../../src/api/db';
+import dbConfig from '../../knexfile';
+
+import { login, createPost } from '../../test-helpers/api';
+import Controller from '../../src/api/controller';

```

Comment:
* first block of imports for system-modules (none here)
* second block for the ones from npm
* third block for local modules

blocks are divided by empty lines

2 empty lines after all imports are finished